### PR TITLE
Prettify markdown, fingerprint autoparsing

### PR DESCRIPTION
**Pull request by TheTechromancer**
Greetings,

This PR uses [prettier](https://prettier.io/) to automatically prettify the markdown table. This is intended to make it easier to read and maintain.

A github workflow has been added which automatically parses the markdown, generates `fingerprints.json`, and commits it to the repo.

image

### CODE REVIEW

File: scripts/gen_fingerprints.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```
Comment (EdOverflow) [inline scripts/gen_fingerprints.py:1]: Thank you for these contributions, @TheTechromancer. Would it be possible to include a comment explaining of this purpose of this file for future maintainers? It's just so to make things easier for others when they want to contribute to the project.

Comment (TheTechromancer) [reply scripts/gen_fingerprints.py:1]: Sure! Note that right now, `fingerprints.json` doesn't have all the information needed for automated detection. There are still a few steps we want to take, including expanding the table to include other fields such as the vulnerable domains involved. But this PR is meant to lay the groundwork, as those steps will take significantly more time and testing.

**Review by EdOverflow — APPROVED** [review]

**Comment by TheTechromancer** [comment]
Update:
- Added a system for auto-verifying fingerprints
- Added colums `Verified` and `Domains` to assist with automated tooling
image

**Comment by EdOverflow** [comment]
Still not too sure about the `Autocheck` column name. I will see if I can come up with a more descriptive column name.

**Comment by pdelteil** [comment]
> This looks good to me. I plan on merging this after I have reviewed @pdelteil's PRs.

The PR's I sent are very simple, and it's been two weeks already.  😢

**Comment by EdOverflow** [comment]
> Still not too sure about the `Autocheck` column name. I will see if I can come up with a more descriptive column name.

@TheTechromancer, would something along the lines of "Verified by CI/CD" be more suitable as a column name? We have to keep in mind that the README has to be presentable to humans.

**Comment by TheTechromancer** [comment]
> @TheTechromancer, would something along the lines of "Verified by CI/CD" be more suitable as a column name? 

That sounds good to me. I'll make the change

**Pull request merged**